OPAL/br/src/main/scala/org/opalj/br/Type.scala

```diff
@@ -188,10 +188,7 @@ sealed trait Type extends UIDValue with Ordered[Type] {
      * Returns the binary name of this type as used by the Java runtime. Basically
      * returns the same name as produced by `Class.getName`.
```

Comment:
```suggestion
     * Returns the binary name of this type as used by the Java runtime.
     * Returns the same name as produced by `Class.getName`.
```
Lets update the comment so it says the intended truth.